tests/kernel/run_ltp.pm

```diff
@@ -0,0 +1,196 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Extracts test commands from an LTP runfile and executes them on the guest.
+# Maintainer: Richard Palethorpe <rpalethorpe@suse.com>
+# More documentation is at the bottom
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils;
+use Time::HiRes qw(gettimeofday);
+require bmwqemu;
+
+sub parse_runfile($$;$) {
```

Comment:
don't use subroutine prototypes, just write `sub parse_runfile {…` . For a reasoning please consult coolo